website/content/en/docs/advanced-topics/scorecard/kuttl-tests.md

```diff
@@ -4,160 +4,226 @@ weight: 50
 ---
 
 This guide outlines the steps which can be followed to implement scorecard
-tests using the [kuttl][kuttl] project and specifically the scorecard
-kuttl test image.
+tests using a [kuttl][kuttl] test suite.
+
+## Defining kuttl Test Cases
+
+On running `operator-sdk create api` for the first time, a kuttl config
+and a set of kuttl [steps][steps] and [asserts][asserts] are scaffolded in `test/kuttl`
+that run a simple install-update-delete test cycle for your new API (and are named as such).
+Let's say you've created an API kind `Memcached`, group `cache`, and version `v1alpha1`.
+The kuttl test files would look like:
+
+```console
+$ tree test/
+test/
+└── kuttl
+    ├── kuttl-test.yaml
+    └── v1alpha1
+        ├── 00-install-memcached.yaml
+        ├── 01-assert-memcached.yaml
+        └── 01-modify-memcached.yaml
+```
+
+Here's a quick overview of what the files do:
+- `test/kuttl/kuttl-test.yaml` - contains a kuttl [`TestSuite` configuration][suite].
+This particular configuration is geared towards running your test cases via scorecard so is quite minimal.
+- `test/kuttl/v1alpha1/00-install-memcached.yaml` - installs a Memcached CR with a minimal spec.
+- `test/kuttl/v1alpha1/01-modify-memcached.yaml` - updates the installed Memcached's spec.
+- `test/kuttl/v1alpha1/01-assert-memcached.yaml` - asserts that the spec update occurred.
+
+Lets take a look at a few of these files, first being `kuttl-test.yaml`:
+
+```yaml
+apiVersion: kudo.dev/v1beta1
+kind: TestSuite
+testDirs:
+- v1alpha1
+# +kubebuilder:scaffold:kuttl:testDirs
```

Comment:
It shows out of the scope of this PR. However, could not we improve the marker`testDirs `? Is it already in use or something new?